src/confirmationModal/index.tsx

```diff
@@ -135,7 +147,13 @@ class ConfirmationModal extends Component<ConfirmationModalProps> {
         <TransitionGroup component="div" className="transition-wrapper">
           {isOpen && (
             <CustomTransition animationName={AnimationType.SLIDE_UP}>
-              <div className={classNames}>
+              <div
+                className={classNames}
+                role="alertdialog"
```

Comment:
alertdialog should be for simple dialog with only text and buttons (like a window.prompt).
I think we should use the role dialog.